djconnectwise/management/commands/cwsync.py

```diff
@@ -181,6 +183,11 @@ def handle(self, *args, **options):
                 self.stderr.write(msg)
                 error_messages += '{}\n'.format(msg)
                 failed_classes += 1
+            except ConnectWiseSecurityPermissionsException as e:
```

Comment:
Because ConnectWiseSecurityPermissionsException is a subclass of ConnectWiseAPIerror, it should probably be handled first. And since CommandError would cause the syncs to fail when you add +1 to the failed_classes, I think within ConnectWiseSecurityPermissionsException you don't want to increment failed_classes.

Reply:
This is resolved.